scripts/templates/docker-compose.template

```diff
@@ -36,6 +36,7 @@ hydroshare:
     POSTGIS_USER: postgres
     PGPASSWORD: postgres
     SOLR_HOST: solr
+    TMP: /hs_tmp
```

Comment:
@aphelionz Also, now that you are at it, can you change this line https://github.com/hydroshare/hydroshare/blob/develop/hydroshare/settings.py#L489 to use /hs_tmp as the temporary directory as well? Thanks.